app/client/pages/course/courseSummary.js

```diff
@@ -0,0 +1,220 @@
+                  display: 'block',
+                  'margin-left': 'auto',
+                  'margin-right': 'auto',
+                  height: '95%',
+                  width: '85%',
+                }} />
+              </Typography>
+
+            </Grid>
+          </Grid>
+
+        </Grid>
+      </Grid>
+      <Container sx={{ 'margin-bottom': '80px' }}>
+
+        <Box sx={{ overflow: 'hidden', minWidth: 250, mt: 1 }}>
+          <Paper variant='outlined' sx={{ 'margin-bottom': 20 }}>
+            <Grid container spacing={2} sx={{ 'margin-bottom': 20, padding: 5 }}>
+
+              <Grid item sx={{ width: '100%' }}>
+                <Typography variant="h4">
+                  Course Description
+                </Typography>
+
+              </Grid>
+              <Grid item sx={{ width: '100%' }}>
+                <Typography>
+                  {post.course_info}
+
+                </Typography>
+                <Grid item sx={{
+                  width: '100%', display: 'flex',
+                  'flex-wrap': 'wrap'
+                }}>
+                  {post.tags.map((tag) => {
+                    return (
+                      <Paper variant='elevation' sx={{ 'margin-right': 7, 'margin-top': 19 }}>
+                        <Typography sx={{ padding: 1 }}>
+                          {tag}
+                        </Typography>
+                      </Paper>)
+                  })}
+                </Grid>
+
+              </Grid>
+            </Grid>
+          </Paper>
+          <Paper variant='outlined' >
+            <Grid container spacing={2} sx={{ 'margin-bottom': 20, padding: 5 }}>
+              <Grid item sx={{ width: '100%' }}>
+                <Typography variant="h4">
+                  Course Chapters
+                </Typography>
+              </Grid>
+              <Grid item sx={{ width: '100%' }}>
+                <Table sx={{ minWidth: 250 }} >
+                  <TableHead>
+                    <TableRow>
+                      <TableCell>
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                        &nbsp;
+                      </TableCell>
+                      <TableCell>Chapter Name</TableCell>
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {post.chapters.map((chapter, index) => {
+                      return (
+                        <TableRow>
+                          <TableCell>{index + 1}</TableCell>
+                          <TableCell>{chapter.chapter_name}</TableCell>
+                        </TableRow>
+                      );
+                    })}
+                  </TableBody>
```

Comment:
There is no need to have chapter indeces. Just a single columns with chapter names will be fine